src/components/HowItWorksDialog.vue

```diff
@@ -14,44 +14,44 @@
         size="x-large"
         data-cy="how-it-works-button"
       >
-        How it works
+        {{ t('home.howItWorks.button') }}
       </v-btn>
     </template>
     <template #body>
       <section class="d-flex flex-column">
-        <span class="my-2">Our weekly play sessions are open to everyone:</span>
-        <li>Wednesday nights at 8:30pm EST</li>
-        <li>Thursdays at 12pm EST</li>
-        <span class="my-2"> Can&#39;t find a match? Join us on 
+        <span class="my-2">{{ t('home.howItWorks.content') }}</span>
+        <li>{{ t('home.howItWorks.wednesday') }}</li>
+        <li>{{ t('home.howItWorks.thursday') }}</li>
+        <span class="my-2"> {{ t('home.howItWorks.findMatch') }} 
           <a 
             class="text-anchor text-decoration-none" 
             href="https://discord.gg/9vrAZ8xGyh" 
             target="_blank"
           >
-            discord 
+            {{ t('home.howItWorks.discord') }} 
```

Comment:
Because discord is proper noun/name I think we actually don't need to translate it. That's one thing that I think it's okay to hard code
```suggestion
            discord
```